transformers/storage/test_helpers/test_helpers.go

```diff
@@ -30,3 +32,38 @@ func CreateHeaderWithHash(hash string, timestamp int64, blockNumber int, db *pos
 	fakeHeader.Id = headerId
 	return fakeHeader
 }
+
+func CreateFakeDiffRecord(db *postgres.DB) int64 {
+	return CreateFakeDiffRecordWithHeader(db, fakes.FakeHeader)
+}
+
+func CreateFakeDiffRecordWithHeader(db *postgres.DB, header core.Header) int64 {
+	fakeRawDiff := fakes.GetFakeStorageDiffForHeader(header, common.Hash{}, common.Hash{}, common.Hash{})
+	storageDiffRepo := repositories.NewStorageDiffRepository(db)
+	diffID, insertDiffErr := storageDiffRepo.CreateStorageDiff(fakeRawDiff)
+	Expect(insertDiffErr).NotTo(HaveOccurred())
+
+	return diffID
+}
+
+func CreateDiffRecord(db *postgres.DB, header core.Header, hashedAddress, key, value common.Hash) utils.PersistedStorageDiff {
+	rawDiff := utils.RawStorageDiff{
+		HashedAddress: hashedAddress,
+		BlockHash:     common.HexToHash(header.Hash),
+		BlockHeight:   int(header.BlockNumber),
+		StorageKey:    key,
+		StorageValue:  value,
+	}
```

Comment:
Can we make use of the helper in vulcanizedb here?

Reply:
fdfe915